lib/puma/state_file.rb

```diff
@@ -1,15 +1,37 @@
 # frozen_string_literal: true
 
-require 'yaml'
-
 module Puma
+
+  # In previous versions of Puma, YAML was used to read/write the state file.
```

Comment:
We should be clearer if this is a refactor or if the state files are incompatible across versions I think

Reply:
There is no change to the state file from past versions.  Since we're just using numeric and string values, writing it with code was QED.  The tests are still using yaml/psych to parse the generated file.

Should I add something stating no changes from previous?

Reply:
I updated the class comments?